### PR TITLE
Chore: Fix kubeconfig error message in backend component

### PR DESCRIPTION
**Pull request by raghavi101**
#### Description:
Fix error message when cluster credentials secret does not have kubeconfig key

#### Link to JIRA Story (if applicable):

https://issues.redhat.com/browse/GITOPSRVCE-400

### CODE REVIEW

File: backend/eventloop/application_event_loop/application_event_runner_test.go [hunk 1]
```diff
@@ -1130,7 +1130,7 @@ var _ = Describe("ApplicationEventLoop Test", func() {
 				},
 				Type: sharedutil.ManagedEnvironmentSecretType,
 				Data: map[string][]byte{
-					"kubeconfig": ([]byte)(kubeConfigContents),
+					shared_resource_loop.Kubeconfig_key: ([]byte)(kubeConfigContents),
```
Comment (anandf) [inline backend/eventloop/application_event_loop/application_event_runner_test.go:1133]: Shouldn't the variables be in camel case ? like `KubeconfigKey` and all small case for the package name `sharedresourceloop` ?

Comment (raghavi101) [reply backend/eventloop/application_event_loop/application_event_runner_test.go:1133]: Thanks Anand, I'll correct it to `KubeconfigKey` although the package name is `shared_resource_loop` and it is being used in multiple places throughout the code

**Review by jgwest — APPROVED** [review]
LGTM too, thanks @raghavi101! And thanks @anandf for reviewing

**Pull request merged**